lib/internal/errors.js

```diff
@@ -232,7 +232,13 @@ E('ERR_NO_ICU', '%s is not supported on Node.js compiled without ICU');
 E('ERR_NO_LONGER_SUPPORTED', '%s is no longer supported');
 E('ERR_PARSE_HISTORY_DATA', 'Could not parse history data in %s');
 E('ERR_SOCKET_ALREADY_BOUND', 'Socket is already bound');
-E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
+E('ERR_SOCKET_BAD_PORT', (port) => {
+  let portMsg = '';
+  if (typeof port !== 'undefined' && port !== null) {
+    portMsg = `:${port}`;
+  }
+  return `Port${portMsg} should be > 0 and < 65536`;
+});
```

Comment:
This is more complicated then it has to be. I would rather should have
```js
E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536. Received %s.');
```

Reply:
fixed (commit: b336aa1)